lang/python.md

```diff
@@ -2,10 +2,154 @@
+
+To convert your code, you must first use a Python shell, which calls upon the interpreter through something called a “bang” line.
+
+As for creating an application or file, there are two ways to do this. You can create a program using a simple text editor like WordPad, or Notepad++. You can also create a program using a Python shell. There are advantages and disadvantages to each method, which we’ll discuss next.
+
+## Python Shell versus Text File :
+A shell is a program or tool that can be used to interact with a system. For instance, the Windows operating system shell can be tapped into by using a “terminal” or command line to submit commands and arguments.
+
+With Python, things work a bit differently than an operating system shell. The Python shell is used to interact with an interpreter, which feeds code to a computer in a form that it can understand.
+
+When you execute a Python program that you’ve written, the interpreter reads the code and converts it into usable commands. The important thing to note is that all of this is done after the program has been executed.
+
+With a shell, the interpreting – or conversion – happens in real-time as you type the code into the computer or system. This means that the actual program is executing as you type. This gives you some idea of how your final code will look, and what your program is actually going to do.
+
+When you write code in a text file, none of that happens until you feed the document into an interpreter. If you have Python installed on your computer you can call upon the interpreter using a command line, but this step is done after you’ve already written the code.
+
+This makes it more difficult to spot errors in your code, and it can also be frustrating if the interpreter runs into issues, because they may not be as apparent as they would if you had used a shell. Still, a lot of developers prefer to use a text editing tool because it is simple and easy to do.
+
+There are text editors with increased functionality – like Notepad++ – which were specifically developed with programming in mind.
+
+## The Best Place to Start :
+efore you can do anything with a programming language, you first need to configure the development environment. Now, we’re going to cover how to setup Python and the interpreter that will execute your custom programs.
+
+Python comes pre-installed on Mac and a majority of Linux distributions. However, you may need to download an updated version depending on how old your system is.
+
+**TIP:** You can easily check your Python version by opening the terminal and running the following command:
+
+```
+    Python -v
+```
+If you’re running Windows, you’ll need to download Python from the Python Software Foundation (link below).
+
+## How Do I Get Python? 
```

Comment:
This & the below section must be included under, `## Setting up Python development environment`, include this at last i.e, L84 & remove this line.

```suggestion

```